kratos/python_scripts/compare_two_files_check_process.py

```diff
@@ -82,9 +82,9 @@ def ExecuteFinalize(self):
         Please see the respective files for details on the format of the files
         """
 
-        KratosMultiphysics.DataCommunicator.GetDefault().Barrier()
+        KratosMultiphysics.Testing.GetDefaultDataCommunicator().Barrier()
```

Comment:
Just for me to know. Shouldn't we take it from the parallel environment in here? (even though this is normally used for the testing we are out of the scope of a test).

Reply:
Usually I would agree with you, but this class derives from Unittest, hence I went for the testing one:
https://github.com/KratosMultiphysics/Kratos/blob/d200e8fd4fdb2c5fead72a04538cfc2d560537fd/kratos/python_scripts/compare_two_files_check_process.py#L19

Reply:
Makes sense for me.

Reply:
probably we should move this file to the `testing` subfolder